src/fa/game_session.py

```diff
@@ -20,7 +20,6 @@ class GameSessionState(IntEnum):
     # Game is running and we're relaying gpgnet commands
     RUNNING = 4
 
-
```

Comment:
2 blank lines before Classes, makes PyCharm happy and:
https://www.python.org/dev/peps/pep-0008/#blank-lines